api/app/models/theq/office.py

```diff
@@ -13,8 +13,8 @@
 limitations under the License.'''
 
 from qsystem import db
-from .base import Base
-
+from app.models.theq import Base
+from app.models.bookings import Exam, Room
```

Comment:
Do we use these imports?

Reply:
![screen shot 2018-11-29 at 10 45 36 am](https://user-images.githubusercontent.com/29988052/49244151-f3d82080-f3c3-11e8-8bd3-5eefd52d3866.png)

Reply:
above is what I get if I try to run the app with the imports commented out

Reply:
and then with them uncommented, the app will run